src/components/shortcut/shortcut.e2e.ts

```diff
@@ -0,0 +1,145 @@
+import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
+
+describe('limel-shortcut', () => {
+    let page: E2EPage;
+    let limelShortcut: E2EElement;
+    let link: E2EElement;
+    let label: E2EElement;
+    let badge: E2EElement;
+
+    describe('with a label', () => {
+        beforeEach(async () => {
+            page = await createPage(`
+                <limel-shortcut label="iSpiffy"></limel-shortcut>
+            `);
+            limelShortcut = await page.find('limel-shortcut');
+            link = await page.find('limel-shortcut >>> a');
+            label = await page.find('limel-shortcut >>> span');
+        });
+        it('displays the correct label', () => {
+            expect(label).toEqualText('iSpiffy');
+        });
+        it('includes the label in `aria-label`', () => {
+            expect(link).toEqualAttribute('aria-label', 'iSpiffy');
+        });
+
+        describe('when changing the label', () => {
+            beforeEach(async () => {
+                limelShortcut.setProperty('label', 'imTired');
+                await page.waitForChanges();
+            });
+            it('displays the new label', async () => {
+                expect(label).toEqualText('imTired');
+            });
+            it('updates `aria-label`', () => {
+                expect(link).toEqualAttribute('aria-label', 'imTired');
+            });
+        });
+    });
+
+    describe('with `linkTitle`', () => {
+        beforeEach(async () => {
+            page = await createPage(`
+                <limel-shortcut link-title="Open the iSpiffy app"></limel-shortcut>
```

Comment:
@LucyChyzhova I pushed a fix for the tests. I had written `linkTitle="…"` here, instead of `link-title="…"` 😄 That's why the tests where failing. Sorry about that!